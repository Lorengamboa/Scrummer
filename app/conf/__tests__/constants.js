const constants = require('../constants');

test('constants should be defined', () => {
  expect(constants).toMatchSnapshot();
});
