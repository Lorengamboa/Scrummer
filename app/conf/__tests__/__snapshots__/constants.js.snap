// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants should be defined 1`] = `
Object {
  "DIR_CREATE_HTML": "/../../app/views/create/create.html",
  "DIR_MAIN_HTML": "/app/views/index/index.html",
  "DIR_MODIFY_HTML": "/../../app/views/modify/modify.html",
  "NAME_DB_NOTES": "notes",
  "SHORCUT_CONSOLE_IOS": "Command+Alt+I",
  "SHORCUT_CONSOLE_WN": "Ctrl+Shift+I",
  "SHORCUT_QUIT_IOS": "Command+Alt+Q",
  "SHORCUT_QUIT_WN": "Ctrl+Shift+Q",
}
`;
